demos/src/scss/_demos.scss

```diff
@@ -48,10 +48,9 @@ body:before {
 }
 
 .o-grid-container {
-	background: lighten(#f6e9d8, 8.5%);
+	background: #EAE8DF;
 }
```

Comment:
Your call, but having spent weeks working on this grid (and Rhys as well), I'd like to say that this is the kind of detail that makes the debugging experience much better and spotting the tiniest mis-alignment much easier.